src/js/media-gesture-receiver.js

```diff
@@ -54,9 +50,19 @@ class MediaGestureReceiver extends window.HTMLElement {
       this._pointerType = e.pointerType;
     };
 
-    this.addEventListener('pointerdown', pointerDownHandler);
+    window.addEventListener('pointerdown', pointerDownHandler);
+
+    // Cannot use composedPath or target because this is a layer on top and pointer events are disabled.
+    // Attach to window and check if click is in this element's bounding box to keep <video> right-click menu.
+    window.addEventListener('click', (event) => {
+      const { clientX, clientY } = event;
+      const { left, top, width, height } = this.getBoundingClientRect();
```

Comment:
idea: could we instead only `getBoundingClientRect()` on `resize` events to avoid recomputing on any page click?

Reply:
Would this be helpful as an alternative as well? https://developer.mozilla.org/en-US/docs/Web/API/Document/elementFromPoint

Reply:
ah, elementFromPoint might not work due to the point-events:none thing